SECURITY_CONTACTS

```diff
@@ -0,0 +1,11 @@
+# Reporting a security vulnerability
+
+Please do not disclose any security issue publicly e.g. Pull Requests, Issues, Comments.
```

Comment:
should mention also not disclosing to project members or OWNERS directly?

Reply:
Done.